### ISSUE TITLE
Consider changing the wifi password

### ISSUE THREAD

**Issue by Helen-8** [DoESLiverpool/somebody-should#1259]
This is just a suggestion and I know that it may well cause some difficulties but I think that we should at least consider changing the wifi password, for a few reasons:

- The current password can be a little niche and alienating (at least one person has mentioned to me they didn't feel massively welcomed by it)
- It would be an option to create a good first impression to people entering the space if we hada funny or very friendly password
- The current password is easy to mistake - people often have to retype a few times before it works

**Comment by johnmckerrell** [comment]
Sounds fine, if we change the password for the existing SSID we'd need to make sure all the devices could be updated, doorbots, sonoff, various ESP32 things etc. An alternative might be to have a new SSID with new password and the stop broadcasting the old one but have it continue to work, I'm not sure what we'd call a new one though as "DoES Liverpool" seems ideal and quite simple.

**Comment by ajlennon** [comment]
If we are going to change things around then I'd be in favour of the new SSID option and breaking things out between sensors and peoples' devices as @MatthewCroughan and @goatchurchprime have been discussing elsewhere.

**Comment by ajlennon** [comment]
I also have terrible terrible trouble with the other password we use for the machines and the variants thereof. fwiw I would love to use a machine password too which is less easily confusable.

**Comment by jackie1050** [comment]
Happy to change, agreed it's difficult to spell - lots of people do seem to find it funny though, it sort of fits in with a creative culture.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by MatthewCroughan** [comment]
This will break and disservice a lot of devices. Why not simply solve #937 for guests and have all of our workshop computers be wired in by ethernet instead so that people don't need to remember passwords for those?

We can set up as many guest networks as we want, can't we? Why change the existing one and create work for ourselves repairing the state of devices that have been relying on this ssid combo? This is exactly why #1217 and #1221 need to consider having their own networks. Because right now, this change would break a lot of things. Whereas if we had separate networks for everything, we can change things like this without causing any breakage.

**Comment by Sean-anotherone** [comment]
Existing password is inspiring and accurate, the place runs on tea & coffee !

**Comment by MatthewCroughan** [comment]
@Sean-anotherone And coke #327 
>  johnmckerrell commented on Aug 18
I bought 3 boxes of Coke Zero a week ago. It’s quite possible Matt has drank them all but are there definitely not some down the side of the fridge?

Deaspartamebad00?

**Comment by ajlennon** [comment]
What about sucrosebad00? That's very of the moment and it should be a good while before society changes enough that anybody is offended by it?

**Comment by MatthewCroughan** [comment]
@ajlennon in that society the password would be good00

**Comment by ajlennon** [comment]
climatechangebad00 would work I imagine?

gretabad16 probably not so much?

**Comment by DoESsean** [comment]
You obviously don't go online at all...

My suggestion is puppiesareevil666. It's so absurd as to be completely
inoffensive to anyone.

**Comment by MatthewCroughan** [comment]
> climatechangebad00 would work I imagine?
> 
> gretabad16 probably not so much?

In 10 years when they look back at your github history, you're going to regreta that statement.

**Comment by ajlennon** [comment]
You leave puppies out of this @DoESsean !!! We LOVE puppies and are not going to have anything said against them.

**Comment by johnmckerrell** [comment]
Can we not start making a big joke of it and seeing what other contentious passwords we can come up with please? 

I'm happy keeping the old SSID and password but "hiding" it so that existing devices don't get kicked offline. I'm thinking "DoES Liverpool Wi-Fi" as the new SSID unless anyone can suggest something better, and we've discussed ******** as the new password with capitals at the beginnings of the words and no spaces although we might change the last word if we can come up with something that flows better.

(I did consider "Do Epic WiFi" for the SSID but then if you don't know what DoES stands for it isn't as obvious as it might seem, but then you still need to find out the password so maybe that's fine)

**Comment by ajlennon** [comment]
I think the gentle point being made here is you can't avoid offending some people some of the time.

And whilst one should (imho) try not to offend in general you shouldn't try to not offend anybody as you end up disappearing up your own tail-pipe.

That said,

"DoES Liverpool Wi-Fi" - this feels really long and the kind of thing we might potentially run into string limit problems with on some devices. "DoES WiFi" maybe?

I hate all this #leet stuff as I can never remember what I'm supposed to be transposing (and also it's tacky) so avoidance of that would be great imho.

**Comment by ajlennon** [comment]
In fact now I come to think about it, in terms of levels of offensiveness I find the use of l33t speak - given historical context - far more offensive than whether somebody likes caffeine in their brew.

**Comment by MatthewCroughan** [comment]
I like decentralization and new networks, and you can set up as many of them as you want. I vote for that, less can go wrong in the future if we have separate networks for things. I know a few people I follow online put all their IOT devices on a separate segregated network for example, two offices on separate networks, etc. :)

**Comment by MatthewCroughan** [comment]
@ajlennon I don't like the spaces, I have no idea how I would put that into wpa_supplicant.conf. Do I need to use %20 now in all my configurations if there are spaces in the password rather than underscores?

**Comment by MatthewCroughan** [comment]
https://www.reddit.com/r/archlinux/comments/3fpa7h/netctl_ssid_has_spaces/

Having spaces in SSIDs I think is a bad idea just because it makes configuration a bit harder.

**Comment by ajlennon** [comment]
>@ajlennon I don't like the spaces, I have no idea how I would put that into wpa_supplicant.conf. Do I need to use %20 now in all my configurations if there are spaces in the password rather than underscores?

Agreed. I was thiinking about this as I suggested it. You actually put then in quotes and it works fine but from a device perspective it's better not to have spaces and "funny" characters.

That said from a human user POV spaces are nice, so #prosandcons ...

**Comment by johnmckerrell** [comment]
The original word used was "alienating", FWIW. If we're trying to get people in through the door and get them to stay we should do anything possible to stop them feeling alienated. It is entirely possible not to alienate, or offend, when setting a WiFi password.

Referring to your point as gentle doesn't make it more relevant or the arguments against less relevant, and it comes across as quite patronising. People saying "it doesn't offend me" doesn't mean it's not true that it alienates others.

Thank you @MatthewCroughan for keeping to the technical side of things.

Yes "DoES WiFi" would be shorter and acceptable although tbh I'm coming around to "Do Epic WiFi" as it gives you the opportunity to explain the meaning of "DoES" (and the findability of DoES Liverpool" is less relevant if you need to the password).

(spaces notwithstanding)

**Comment by ajlennon** [comment]
>@ajlennon The original word used was "alienating", FWIW. If we're trying to get people in through the door and get them to stay we should do anything possible to stop them feeling alienated. It is entirely possible not to alienate, or offend, when setting a WiFi password.

Let's have a vote between members of the space then. (a) Who finds it alienating? (b) Who thinks this password would alienate new visitors to the space?

**Comment by ajlennon** [comment]
Add it to the next community meeting and lets not descend into a conversation where you accuse me of being patronising (as we all want to steer away from that kind of ad hominem language I am sure)

**Comment by MatthewCroughan** [comment]
This poll allows comments and is a simple yes or no to changing the password. Only one vote per IP and one choice.

https://strawpoll.com/eysgs23b

**Comment by johnmckerrell** [comment]
"one vote per IP" 🤔

**Comment by MatthewCroughan** [comment]
@johnmckerrell Yes, as in without checking that option, you could just use an incognito window and it only verifies cookies. Yes, I could just login and vote 5 times on my 5 ips, but I'm not going to do that, it's just to get a gauge.

**Comment by johnmckerrell** [comment]
@MatthewCroughan but we're all on a single IP here!

**Comment by MatthewCroughan** [comment]
@johnmckerrell I have made a brutal miscalculation and I've ordered the guillotine for my execution, it'll arrive in an amazon prime package tomorrow.

**Comment by ajlennon** [comment]
A simple raising of hands at a meeting that community members can be bothered to attend to have their voice heard would be more than adequate imho.

**Comment by MatthewCroughan** [comment]
@ajlennon Agreed, I've updated the link to not do ip verification. That was funny :dancer:

**Comment by Helen-8** [comment]
I'm happy to leave it to the next Community Meeting if you want. I originally suggested the github issue as "considering" changing the password because I have had negative feedback from new people in the past (not to mention the clumsiness of the spelling), but didn't want to create a storm in a teacup (which I don't think I've managed to do!).

However, to make my reasoning more explicit, caffeine can make people anxious, have heart palpitations and other effects and I think that DoES being the awesome community that it is could have a password which better reflects DoES, especially if it doesn't mean a massive change logistically. When people just arrive, pitch up, and the first conversation with them is about the wifi, it's a good opportunity to make the conversation as open as possible.

**Comment by johnmckerrell** [comment]
@ajlennon I did not intend to attack your character, rather suggest that your choice of words may have given a meaning you didn't intend.

**Comment by ajlennon** [comment]
>@ajlennon I did not intend to attack your character, rather suggest that your choice of words may have given a meaning you didn't intend.

All I would say is that we should all remember that words which you may view as offensive may well be viewed by others as offensive. And I certainly view a director of the space suggesting in a public forum that I am patronising as both alienating and offensive. Strange, as this whole conversation is about how not to alienate people. I do think that ad hominems should be viewed as unacceptable in forum discussion to be honest...

**Comment by johnmckerrell** [comment]
"we should all remember that words which you may not view as offensive may well be viewed by others as offensive"
@ajlennon does it matter that I found your "gentle" to be offensive to me? I felt like you were using that word to patronise me. I may have been wrong in that belief but as you're saying we should be more careful what words we use would that not apply to you also? For the record I don't agree with your assertion that saying that some of the language you used "came across as quite patronising" was an ad hominem attack.

**Comment by ajlennon** [comment]
>@ajlennon does it matter that I found your "gentle" to be offensive to me?

I referrred to a gently made point. You may agree or disagree but I said nothing about you, your comments, or your attitudes. I try to avoid ad hominems as they are inflammatory and unhelpful.

I would appreciate the same level of respect from you in your public role in public forums in future.

**Comment by ajlennon** [comment]
On that note this conversation seems to have moved from a reasoned discussion to something I am finding personally upsetting so I'm going to step back and take no further part in it.

**Comment by johnmckerrell** [comment]
@ajlennon I am quite happy to show you respect both in public and private! I asserted that your language came across as patronising to me, you have corrected me, I am happy to accept that this was not your intention. Can we consider this resolved?

**Comment by MatthewCroughan** [comment]
I genuinely believe there are a class of issues the are best suited for community environments, I've noticed in general that subjective issues like this aren't handled well on Github, because the issue list is a more logical conclusive, and this issue is more subjective than objective.

Making matters worse, the github issues list is not necessarily the most far reaching thing does has, the google group and the physical community meeting is likely to reach a wider range of people. Although I'm glad you posted @Helen-8 otherwise then the techies wouldn't know about it either. Maybe there needs to be a "community" issue label to identify issues that are more subjective and community based as this one is.

That said, my opinion is that it's just a password/ssid, which is why I feel this is best solved with a new network rather than an argument or even a discussion since they aren't complicated, don't have implications and require no thought to add to the existing stack of networks we have.

**Comment by ajlennon** [comment]
>Can we consider this resolved? 

Absolutely buddy. And apologies if you felt I was patronising you.

**Comment by seanspotatobusiness** [comment]
I don't find the current SSID offensive to me but I can see why some
people, who are already in a minority in drinking decaffeinated drinks,
would feel alienated by it. Even if most people are not offended by it and
therefore don't see the need for a change, I don't see the harm in creating
another SSID and password for guests?

**Comment by johnmckerrell** [comment]
@MatthewCroughan unfortunately the physical community meeting probably has the smallest reach and probably isn't very representative of the whole community. I thought the reach of GitHub might be bigger than you think, although who knows how many people actually have the notifications turned on.

**Comment by MatthewCroughan** [comment]
@seanspotatobusiness that's why I was suggesting earlier we just solve #937 and consider it done.

**Comment by MatthewCroughan** [comment]
@johnmckerrell Well I'm just going off some of @Helen-8's points at Future Gazing that I recall regarding the techie/co-worker divide, as well as responses generally I've gotten that suggest that Github is niche, or that the google groups is where more people are participating. Mike and Jackie in particular always suggest to me in person that if I'm going to do anything regarding fundraising to do it on the google groups for example.

**Comment by johnmckerrell** [comment]
@MatthewCroughan who is the non-sandboxed network for and what is the password on it? I'm quite sure we've had more problems caused by people who have been here a while than by newcomers (and I don't say that to attack anyone, it just seems to be the case, unsurprisingly really as people who are here longer have more time to cause problems!)

**Comment by MatthewCroughan** [comment]
@johnmckerrell I don't know what you mean by problems? What problems have been caused regarding these VLANs? Am I misunderstanding you?

**Comment by johnmckerrell** [comment]
@MatthewCroughan Off the top of my head I can only really think of 3 instances
1. The problem mentioned in #937
2. The time someone caused a loopback in the network (actually this has happened twice maybe?)
3. The time someone plugged in a router that then tried to act as a DHCP server

Only the first was caused by new external users.

**Comment by MatthewCroughan** [comment]
@johnmckerrell Right, well to mitigate that can be done with managed switches and flood protection as an option, but that's a hardware issue. So issue 2 and 3 you've listed could be eliminated with different hardware. I do see incredibly cheap gigabit managed switches from time to time so I'll try to pick one up and donate it to the space.

**Comment by Sean-anotherone** [comment]
It's a bit off topic but I spotted Pepsi max normal & cherry at £6.25 
for 24 tins in Iceland earlier today.   Does that qualify as a Coke 
surrogate?  < ducks and quickly exits the chat>

Sean

**Comment by MatthewCroughan** [comment]
@Sean-anotherone surrogate it is.

**Comment by MatthewCroughan** [comment]
Me and @Helen-8 earlier discussed potential wifi names. Out of that came a few suggestions. But the one I favour is

SSID: DoEpicWifi
Pass: DoEpicWifi

**Comment by DoESsean** [comment]
If, as seems likely, the easiest thing to do is to create a new 'guest' WiFi, can I suggest we just do that?

Would it be @MatthewCroughan or @johnmckerrell that would do it?

Password: Welcome

It's literally the most welcoming thing it could be, doesn't have any possible weird spellings, and doesn't have multiple capitals in the middle.

**Comment by johnmckerrell** [comment]
It’ll need to be me as it should be someone in the space to make sure the old ssid still works ok.

--

**Comment by DoESsean** [comment]
Consider yourself assigned.

**Comment by skos-ninja** [comment]
@johnmckerrell just to note that marking an SSID as hidden means that some devices will not connect to it by default even if the network still exists as per the wifi spec

**Comment by johnmckerrell** [comment]
Ah.. thanks for that Jake. Definitely one to test then!

--

**Comment by chris-does** [comment]
I was reading this thread earlier and tried to give some thought to what might be both acceptable, and if at all possible appealing, combination of WiFi Network Name & Password 
(at least in the context of a "public" / "guest" network - it crossed my mind separately that perhaps some of DoES administrative infrastructure etc. perhaps may benefit from being an purposefully segregated network which I believe would be a conversation for another thread).

This led me to the following:

Network Name (SSID): WelcomeToDoes
Password: CreateInnovateRelate

I am not sure this is objectively (or subjectively) better (or worse) than any of the above suggestions, though I think it does convey some of the themes and values that - as a newcomer to the DoES Community - I have both experienced and observed.

I did consider two variants of the above password;
the first was the same but all in lower case,
the second added "L3" to the end such that it satisfied the requirement of having a number in the password: CreateInnovateRelateL3

I would welcome any thoughts and comments on this and certainly hope this may benefit rather than hinder the resolution of this topic :-)
